Source/Common/Command/CustomValues/CustomValuesCommandArgument.cpp

```diff
@@ -40,6 +42,14 @@ CustomValuesCommandArgument::CustomValuesCommandArgument(const String &name, Par
 		editable->hideInEditor = true;
 	}
 
+	if (enablePrecison && param->type == Controllable::INT)
+	{
+		sendPrecision = addEnumParameter("Precision", "Type of int sent to the device, determine the number of bytes sent for this variable", true);
+		sendPrecision->addOption("4 bytes (Int32)", IntType:: INT32)->addOption("2 bytes (Int16)", IntType::INT16)->addOption("1 byte", IntType::BYTE);
+		//sendType->addOption("uInt32", IntType::UINT32)->addOption("uInt16", IntType::UINT16)->addOption("Char (signed Byte)", IntType::CHAR);
+		//sendType->hideInEditor = true;
```

Comment:
This should probably decomented considering that _editable_ and _param_ are set with "hideInEditor", but when enabling it the parameter doesn't show in the UI (which seems logical considering the name "hideInEditor"). That propably means that _editable_ and _param_ are set back to not hidden but I have no idea where